src/Symfony/Component/ExpressionLanguage/Token.php

```diff
@@ -54,7 +54,7 @@ public function __toString()
     /**
      * Tests the current token for a type and/or a value.
      *
-     * @param array|int   $type  The type to test
+     * @param string      $type  The type to test
```

Comment:
Should be `array|string` as you can pass an array of types as a first argument.

Reply:
How is this supposed to work? A `Token` can only be constructed with a `string` type, so the strict comparison with an array will always be `false`.

Reply:
Sorry, I was confused as Twig allows an array, but not here.